v2/azure-arm.yaml

```diff
@@ -449,6 +449,11 @@ exportFilters:
     version: v*api20210301
     name: RedisEnterpriseDatabase
     because: "including redis enterprise resources"
+  - action: include-transitive
+    group: containerregistry
+    version: v*api20190501
+    name: Registry
+    because: "including container registry resource"
```

Comment:
When I merge PR #2071, you'll end up with a conflict here in the configuration file - we're removing the export filters section in favour of a simpler technique.

You'll instead need to add this resource to the `ObjectModelConfiguration` section; it'll look something like this:

``` yaml
ContainerRegistry:
  2021-09-01:
    Registry:
      $export: true
```

See PR #2080 for the matching updates to _Adding a new code generated resource to ASO v2_.

Reply:
Thanks!